content/blog/standup.md

```diff
@@ -0,0 +1,212 @@
+Identify how many cards should have been moved and current status.
+
+Any tool that publishes data is good enough.
+
+Many teams don't even use any! (These are very mature teams. I would recommend having some visual representation.)
+
+### What not to do
+
+##### Too granular
+
+"I worked on the order class and implemented that method", "I worked on liquibase scripts", nobody cares and should care about that.
+Make it abstract enough as if the recipients are non devs.
+The meeting is for the project, not yourself. Don't go too technical.
+
+Focus on the task or story's completeness or challenge.
+
+This is one of the reasons the meeting becomes boring.
+Because nobody understands what the other person really means, and then they stop paying attention to the other person.
+
+##### Cutting each other
+
+Only one person speaks!
+
+This is one reason why the meeting continues after the designated time and lasts for even an hour!
+
+##### Everyone addressing to one person
+
+The meeting is not a project status report meeting! It is a sync up call.
+Everyone should address the team, not the team lead or project manager.
+
+Non devs or attendees who are not directly working with the team should be quiet or give their updates to
+everyone like any other person but never be a receiver.
+
+##### Slip in other thing.
+
+"Let me show the demo of what I did", "Hey, if you are facing this issue, then do this to solve it", "Why don't you do
+this?", "Why don't we do this?".
+None of this should happen. Everyone should focus on only strictly updates. Nothing else.
+
+Did I say nothing else?
+
+If you want another conversation, meet after the meeting and have a separate meeting with a smaller group.
+Even if the next meeting will have the same participants, don't do it now; call another meeting and do it separately.
+
+##### Q: But we work only remotely, and this is the only time the whole team is together; I would want it to last a little longer and combine other things.
+
+Even in that case, keep the standup time shorter. Be on the same call with the same people, just complete standup in a short time.
+Don't slip in a demo along with standup. Do other things after officially concluding standup. Wear a different hat after that. 
+
+Remember, the standup should be completed on time in a short duration. 
+The meeting or call can be extended with the same or different participants.
+
+##### Too many update
+
+"I worked on setting up that server then talked to Bob then talked to Yoda about order detail then talked to Groot about
+invoice preparation" — How does this help the project? Why should anyone care?
+
+##### Absence of context
+
+"I worked on ABC-549 and ABC-588 and will keep working on that". While this looks like good enough. However, not
+everyone would have context on what "ABC-549" is; instead, try to say, "I worked on order detail screen".
+
+##### Not using burndown chart or team board because it is not updated
```

Comment:
I have seen very bad standup where a "scrum master" will go through all the issues in the board and ask everyone for a status. We should add this as an anti-pattern. People who are going to talk should take the charge of their own work. From updating tickets to delivering it and updating others in the team about it

Reply:
It was already mentioned in a section that
"I disagree with the concept of having a Scrum master. The word master is terrible anyway. 
It also brings all the micromanagement, like tracking the burndown chart like a hawk."

Still I will add it in the same section for more clarity and example.